pkg/k8shandler/kibana/reconciler.go

```diff
@@ -181,6 +190,27 @@ func (clusterRequest *KibanaRequest) deleteKibana5Deployment() error {
 	return nil
 }
 
+func (clusterRequest *KibanaRequest) isClusterLoggingOperatorCreated() bool {
+
+	kibanaCR := clusterRequest.cluster
+
+	if kibanaCR.OwnerReferences != nil {
+		// also check for the owner ref being clusterlogging/instance
+		for _, ref := range kibanaCR.OwnerReferences {
+			if ref.Kind == expectedCLOKind && ref.Name == expectedCLOName {
+				return true
+			}
+		}
+	}
+
+	// this is a secondary check to allow for stand-alone EO testing to work
+	if kibanaCR.Name == expectedCLOKibana && kibanaCR.Namespace == expectedCLONamespace {
+		return true
+	}
```

Comment:
This looks like a test smell. Why not declaring the CLO OwnerReferences on the KibanaCR in our tests directly?

Reply:
because the scheduler would remove it if the CL object isn't there :)
let me think on this a bit more.. because we could just create a cluster logging object as part of this.. but then we're also getting into needing to be able to deploy the crd and that seems like major scope creep for this operator (even for testing)